style.css

```diff
@@ -69,3 +123,295 @@ h1 span {
   margin-top: 10px;
   margin-left: 97px;
 }
+
+.btn {
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  margin-bottom: 20px;
+}
+
+.container2 .indicator {
+  width: 48px;
+  height: 4px;
+  background: #36b37e;
+  border-radius: 24px;
+  display: flex;
+  align-items: center;
+  margin-left: 160px;
+  margin-right: 135.5px;
+  margin-top: 5px;
+}
+
+.container2 h2 {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-size: 40px;
+  font-weight: 700;
+  margin-left: 78px;
+  color: #172b4d;
+}
+
+.first-img {
+  background: url(images/Snapshoot\ Portfolio.png) no-repeat;
+  width: 695vh;
+  height: 220px;
+  margin-left: 10%;
+  margin-top: 50px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+.container2 h3 {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-size: 24px;
+  color: #3a4a42;
+  margin-left: 80px;
+  padding-top: 10px;
+  margin-bottom: 10px;
+  line-height: 32px;
+}
+
+.container2 ul {
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.container2 a button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 15px;
+  margin-left: 115px;
+  margin-bottom: 48px;
+  margin-top: 48px;
+  color: #fff;
+  border: #fff;
+  gap: 10px;
+  width: 122px;
+  height: 48px;
+  bottom: 8px;
+  background: #36b37f;
+  border-radius: 8px;
+}
+
+.container2 a button:hover {
+  color: #1e488f;
+}
+
+.container2 .second-img {
+  background: url(images/snapa.png) no-repeat;
+  width: 295px;
+  height: 220px;
+  margin-left: 10%;
+  margin-top: 0;
+}
+
+.container2 .third-img {
+  background: url(images/Snapb.png) no-repeat;
+  width: 295px;
+  height: 220px;
+  margin-left: 10%;
+  margin-top: 0;
+}
+
+.container2 .fourth-img {
+  background: url(images/Snapc.png) no-repeat;
+  width: 295px;
+  height: 220px;
+  margin-left: 10%;
+  margin-top: 50px;
+}
+
+.container2 .fifth-img {
+  background: url(images/snapa.png) no-repeat;
+  width: 295px;
+  height: 220px;
+  margin-left: 10%;
+  margin-top: 0;
+}
+
+.container2 .sixth-img {
+  background: url(images/Snapb.png) no-repeat;
+  width: 295px;
+  height: 220px;
+  margin-left: 10%;
+  margin-top: 0;
+}
```

Comment:
- [ ]  Your website is not responsive enough please kindly consider to use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels this will help your website responsive on different mobile devices.The website should be responsive, following these breakpoints:

Minimum mobile screen size should be 375px
- [ ] see screen record of your website


https://github.com/AhmedMh0/Mobile-Portfolio/assets/92458236/fe725706-f27b-462e-a455-3650f0c0b248